var/httpd/htdocs/skins/Agent/default/css/Core.Form.css

```diff
@@ -704,26 +704,67 @@ fieldset.TableLike > .Row.MultiColumn > .FieldCell > .Field {
     width: 100%;
 }
 
-/* TODO: Temporary, improve */
-fieldset.TableLike fieldset.DynamicFieldSet > .Field,
-fieldset.TableLike fieldset.DynamicFieldSet > .Row > .FieldCell > .Field {
-    line-height: 1.9em;
-    margin-left: 30%;
-    margin-right: 0;
-    padding-top: 3px;
-    padding-bottom: 3px;
-    padding-left: 10px;
-    padding-right: 0px;
-    min-height: 20px;
+/* Dynamic Field Set */
+
+.DFSetOuterField {
+    max-width: 622px; 
 }
 
-fieldset.TableLike fieldset.DynamicFieldSet > .Row.MultiValue > .FieldCell > .Field {
-    display: inline-block;
+.DynamicFieldSet {
```

Comment:
Be careful, I'm not sure, whether this affects other places like TicketZoom, ConfigItemZoom etc. I would keep the fieldset.TableLike structure here, and maybe add a .Row somewhere outside, too.